ranger/src/chain-ranges-verification-db.ts

```diff
@@ -0,0 +1,47 @@
+import { Config } from "arangojs/connection";
+import { Database, aql } from "arangojs";
+import { toU64String } from "./u64string";
+
+export class ChainRangesVerificationDb {
+    readonly config: Config;
+    readonly database: Database;
+
+    constructor(config: Config) {
+        this.config = config;
+        this.database = new Database(config);
+    }
+    
+    async get_summary(): Promise<ChainRangesSummary> {
+        const cursor = await this.database.query(aql`
+            FOR doc IN chain_ranges_verification
+                FILTER doc._key == "summary"
+                RETURN doc
+        `);
+        const result = await cursor.next() as ChainRangesSummary | null;
+        if (!result) {
+            throw new Error("Chain ranges summary not found");
+        }
+        return result;
+    }
+
+    async update_verifyed_boundary(veryfied_master_seq_no: number): Promise<void> {
+        const chain_order_boundary = toU64String(veryfied_master_seq_no + 1);
+
+        await this.database.query(aql`
+            UPDATE "summary" 
```

Comment:
Maybe use UPSERT here?

Reply:
The ranger requires existing summary to start. So insert is impossible by design.